.ci/azure-pipelines-main.yml

```diff
@@ -55,6 +55,13 @@ jobs:
           packageType: sdk
           version: ${{ parameters.DotNetSdkVersion }}
 
+      - task: SonarCloudPrepare@1
+        displayName: 'Prepare analysis on SonarCloud'
+        inputs:
+          SonarCloud: SC
+          organization: 'jellyfin'
+          projectKey: 'jellyfin_jellyfin'
```

Comment:
I also made guesses on what these two values would be, maybe they're correct already but I don't know

Reply:
I guess we'll see when you change the service connection name. It'll error out soon enough. It looks sane to me.